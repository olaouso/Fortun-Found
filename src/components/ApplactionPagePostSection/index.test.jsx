import React from "react";
import renderer from "react-test-renderer";
import PostSection from "./";

it("renders PostSection correctly", () => {
  const tree = renderer.create(<PostSection />).toJSON();
  expect(tree).toMatchSnapshot();
});
