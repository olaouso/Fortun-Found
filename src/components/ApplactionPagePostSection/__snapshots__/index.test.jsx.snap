// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PostSection correctly 1`] = `
<div
  className="postApplicationPage container"
>
  <div
    className="row"
  >
    <div
      className="col-md-6"
    >
      <img
        alt="post"
        className="postImageApplicationPage"
      />
    </div>
    <div
      className="postTitleDescription col-md-6"
    >
      <div>
        <h2
          className="postTitle"
        />
        <p
          className="postDescription"
        />
        <p
          className="postDescription"
        >
          applicationPageStep1.date
           
        </p>
      </div>
    </div>
  </div>
</div>
`;
